src/ios/BranchSDK.m

```diff
@@ -77,19 +83,8 @@ - (void)initSession:(CDVInvokedUrlCommand*)command
         CDVPluginResult *pluginResult = nil;
         if (!error) {
             if (params != nil && [params count] > 0) {
-                NSError *err;
-                NSData *jsonData = [NSJSONSerialization dataWithJSONObject:params
-                                                                     options:0
-                                                                     error:&err];
-                if (!jsonData) {
-                    NSLog(@"Parsing Error: %@", [err localizedDescription]);
-                    resultString = [NSString stringWithFormat:@"Parsing Error: %@", [err localizedDescription]];
-                    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:resultString];
-                } else {
-                    NSLog(@"Success");
-                    resultString = [[NSString alloc] initWithData:jsonData encoding:NSUTF8StringEncoding];
-                    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:params];
-                }
+                NSLog(@"Success");
+                pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:params];
             } else {
                 NSLog(@"No data found");
                 resultString = @"No data found";
```

Comment:
We should have line 91 (in the new file) use messageAsBool:FALSE (explanation below)